db/migrate/20220514114151_create_follows_relationships.rb

```diff
@@ -0,0 +1,10 @@
+class CreateFollowsRelationships < ActiveRecord::Migration[6.1]
+  def change
+    create_table :follows_relationships do |t|
+      t.references :follower, foreign_key: { to_table: :users }, :null => false
+      t.references :followed, foreign_key: { to_table: :users }, :null => false
```

Comment:
シンボルがキーの場合はコロンを使って書きましょう。

```suggestion
      t.references :follower, foreign_key: { to_table: :users }, null: false
      t.references :followed, foreign_key: { to_table: :users }, null: false
```